compaction/compaction.cc

```diff
@@ -614,6 +617,10 @@ class compaction {
     // Default range sstable reader that will only return mutation that belongs to current shard.
     virtual flat_mutation_reader_v2 make_sstable_reader() const = 0;
 
+    virtual flat_mutation_reader_v2 make_sstable_reader(compaction_task_stats_ptr) const {
```

Comment:
please FIXME comment in this patch ("scrub compaction: register validation errors and fixes in metrics")
with the intent to use the compaction_task_stats_ptr,
since it's unused at this point.